hbp_knowledge/adnp/van-dijck2019.bel

```diff
@@ -76,69 +76,98 @@ SET Citation = {"PubMed", "29724491"}
 SET Evidence = "Fifty-two percent of the individuals in this cohort present with
 severe intellectual disability at the age of assessment, 36% have a moderate
 disability and 12% have a mild disability. "
+SET Confidence = "High"
 p(HGNC:ADNP) -| path(MESH:"Intellectual Disability")
+UNSET Confidence
 
 SET Evidence = "Developmental delay is present in all individuals, with motor
 delay being one of the key features."
-p(HGNC:ADNP) -| path(MESH:"Developmental Disabilities")
```

Comment:
They talked about loss-of-function mutations of the protein that lead to these diseases.
Therefore I coded it the other way around.
So if no normal protein in present children show Developmental Disabilities, that means if the WT protein is present Developmental Disabilities are downregulated.
Therefore the statements initially made should be correct